projects/ngx-datatable/src/lib/components/body/body.component.ts

```diff
@@ -59,7 +68,7 @@ import { translateXY } from '../../utils/translate';
           [groupedRows]="groupedRows"
           *ngFor="let group of temp; let i = index; trackBy: rowTrackingFn"
           [innerWidth]="innerWidth"
-          [ngStyle]="getRowsStyles(group)"
+          [ngStyle]="getRowsStyles(group, indexes.first + i )"
```

Comment:
Here is some magic that does not seem to be obvious and we may need some docs. So we iterate through the rows but have not loaded because data is not available, right? 

Why to we need to call this function at all?

Reply:
It is basically used for get correct position for ghost cell rows. I added detail comments on `getRowsStyles ` function

Reply:
`ngx-datatable` uses css translate to position each row in correct order when doing scroll. It takes the actual row and finds its position based on that. since in ghost rows we actually don't have real rows we pass just the index to get their position